Sources/Stevenson/JiraService.swift

```diff
@@ -96,110 +99,39 @@ extension JiraService {
 
     public func create<Fields>(issue: Issue<Fields>, on container: Container) throws -> Future<CreatedIssue> {
         let fullURL = URL(string: "/rest/api/3/issue", relativeTo: baseURL)!
-        return try container.client()
-            .post(fullURL, headers: self.headers) {
-                try $0.content.encode(issue)
-            }
-            .catchError(.capture())
-            .flatMap {
-                try $0.content.decode(CreatedIssue.self)
-            }
-            .catchError(.capture())
-    }
-}
 
+        let logMessage = "Creating a new issue <\(issue.fields.summary)> on board #\(issue.fields.project.id)"
+        self.logger.info("[JIRA] \(logMessage)")
 
-// MARK: Jira Common Field Types
-
-extension JiraService {
-    public enum FieldType {
-        public enum TextArea {
-            // See spec at: https://developer.atlassian.com/cloud/jira/platform/apis/document/structure/
-            
-            public struct Document: Content {
-                let type = "doc"
-                let content: [DocContent]
-                let version = 1
-                public init(content: [DocContent]) {
-                    self.content = content
-                }
-                public init(text: String) {
-                    self.content = [DocContent.paragraph([.text(text)])]
-                }
+        return try container.make(SlowClient.self)
```

Comment:
Same, using the `SlowClient` here to send those requests serially in hopes of reducing the used quotas and avoid rate-limiting.